README.md

```diff
@@ -1,38 +1,52 @@
-# sv
+# Newsletter template
 
-Everything you need to build a Svelte project, powered by [`sv`](https://github.com/sveltejs/cli).
+A modern template designed for newsletter a mobile-friendly interface and an integrated flow for verifying the sign-up
+emails.
 
-## Creating a project
+## Showcase
 
-If you're seeing this, you've probably already done this step. Congrats!
+A simple and sleek UI allowing users to sign up to your newsletters.
 
-```bash
-# create a new project in the current directory
-npx sv create
+## Highlight
 
-# create a new project in starter
-npx sv create starter
-```
+## Useful links
 
-## Developing
+- Env setup: [.env.example](.env.example), rename to `.env`
+- Site Header: [src/routes/Header.svelte](src/routes/Header.svelte)
+- Site footer: [src/routes/Footer.svelte](src/routes/Footer.svelte)
+- Email template: [src/lib/server/templates/confirm.ejs](src/lib/server/templates/confirm.ejs)
+- Logo: [static/logo.svg](./static/logo.svg) and [static/favicon.png](./static/favicon.png)
+- OG image (preview thumbnail): [static/newsletter-og.png](./static/newsletter-og.png)
 
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+## Technologies
 
-```bash
-npm run dev
+We use [Svelte](https://svelte.dev/docs/svelte/overview) and [SvelteKit](https://svelte.dev/docs/kit/introduction) as a
+web framework with [TypeScript](https://www.typescriptlang.org/) codebase.
 
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
-```
+CSS framework [TailwindCSS](https://tailwindcss.com/) is used for all styling needs.
 
-## Building
+Financial and payments infrastructure [Stripe](https://stripe.com/en-cz) to collect (not only) card payments, generate
```

Comment:
Is this project using Stripe? If not we should remove this
```suggestion
```

Reply:
aah ya correct, leftover after a copy paste haha.